### DISCUSSION TITLE
Majora's Mask - Korean?

### DISCUSSION THREAD

**Discussion by kyul0706** [leoetlino/project-restoration#191] [Q&A] [answered]
Hi, first of all, thank you for the patches. I am looking forward try this out. I was wondering if there is a way to make this work on Korean version of MM3DS? If there is, that would be wonderful. 

Thanks,

**Comment by leoetlino** [comment]
It should work with the Korean version -- just make sure to use the Korean title ID. If you find out what that ID is and if the game works properly please let me know so I can add the ID to the website :)

**Reply by kyul0706** [reply]
Hello, thanks for the reply! This is the title id for the Korean version: 

0004000000185D00

Unfortunately, I was not able to get it to work. It just stays at the initial boot up screen (Nintendo 3DS with red silky lines on top screen). I've tried all three versions (v100, v101, v110). I've tried with Japanese and NA copies, which worked well with no problem. I am not sure what is preventing it from working. If you don't mind, any suggestions would be great!

**Reply by leoetlino** [reply]
Hmm, then it's possible that the executable of the Korean version is different from all the other versions... Can you send me a DM on IRC (leoetlino @ libera.chat) or Discord (leoetlino#1857)?

**Comment by leoetlino** [comment, marked as answer]
An update on this: unfortunately the Korean version actually has new code for supporting Korean message files, so just using the global v100/v101/v110 executables isn't an option. (It does allow the game to boot, but all Korean text is broken.) Because of this, the Korean version cannot be supported at this time. People who have the Korean version will need to play a different version.
